ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,112 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def get_drpc_name(namespace):
+    """
+    Get the DRPC Resource Name
+
+    Args:
+        namespace (str): Name of namespace
+
+    Returns:
+        str: DRPC resource name
+
+    """
+
+    drpc_obj = ocp.OCP(kind=constants.DRPC, namespace=namespace,).get()[
+        "items"
+    ][0]
+    return drpc_obj["metadata"]["name"]
+
+
+def get_drpolicy_name():
+    """
+    Get DRPolicy Name
```

Comment:
@prsurve , same question here as well, is there any chance of having more than one DRPolicy? @am-agrawa , I guess we had same discussion, right?

Reply:
yes, we have can multiple DR Policies applied to a set of workloads (say 20) for testing.

Reply:
i'm getting drpolicy based on namespace where drpc is created